TSC-Charter.md

```diff
@@ -81,21 +81,31 @@ Subject to such policies as may be set by the Board, the TSC is
 responsible for all technical development within the Node.js Foundation,
 including:
 
-* Setting release dates.
-* Release quality standards.
-* Technical direction.
-* Project governance and process (including this policy).
-* GitHub repository hosting.
-* Conduct guidelines.
-* Maintaining the list of additional Collaborators.
-* Development process and any coding standards.
+* Setting release dates
+* Release quality standards
+* Technical direction
+* Maintaining the list of additional Collaborators
+* Development process and any coding standards
 * Mediating technical conflicts between Collaborators or Foundation
-projects.
+projects
+* Management of the `nodejs-private` GitHub organization
 
 The TSC will define Node.js Foundation’s release vehicles and serve as
 Node.js Foundation’s primary technical liaison body with external open
 source projects, consortiums and groups.
 
+The TSC shares responsibility with the Node.js Community Committee ("CommComm")
+for:
+
+* Management of the `nodejs` GitHub organization
+* Project governance and process
+* Conduct guidelines
+
+TSC and CommComm will use the `nodejs/admin` GitHub repository to manage shared
+responsibilities. In the event the CommComm and TSC are unable to agree on a
+decision, the result is _status quo_. Such issues may subsequently be escalated
+to the Board for a final decision. 
```

Comment:
I'm not sure the board is signed up for being the tie breaker, but I don't have a better suggestion so I'm ok as long as the board approves.

Reply:
I agree that it would be nice if there was a non-Board option. On the other hand, this just formalizes what is already the case. If TSC and CommComm disagree, nothing changes. If someone wants to appeal to a higher authority, it's the Board or nothing. So, even if we haven't written it down until now, this accurately describes the situation and doesn't really introduce anything new.

Reply:
Does this represent the current, informal, situation? I do not think so. I would prefer to keep the board out of our decision making process.

Reply:
> Does this represent the current, informal, situation? I do not think so. 

OK, so in your view, what *is* the current process should the following situation arise: TSC and CommComm disagree on a policy change being debated in the admin repo. TSC wants one thing, and CommComm wants something else. Both agree that the status quo is unacceptable. An impasse is reached. Neither group will compromise on what they say the new policy must be. What happens now?

I'm of the opinion that such a situation would be escalated to the board. I would hope we never get to such a situation. But should it arise, I'm unaware of any other resolution mechanism.

Reply:
> TSC and CommComm disagree on a policy change being debated in the admin repo. TSC wants one thing, and CommComm wants something else. Both agree that the status quo is unacceptable. An impasse is reached. Neither group will compromise on what they say the new policy must be. What happens now?

As per the admin repo governance, status quo remains, see https://github.com/nodejs/admin#governance-and-current-members.

I'm -1 on giving any power to the board on our internal governance. The TSC charter was written in this specific way to grant the project the largest autonomy. We should not relinquish that. 

cc @rvagg